### ISSUE TITLE
Accessibility Violations In Line Chart

### ISSUE THREAD

**Issue by Veeresh-Bushetti** [carbon-design-system/carbon-charts#952]
#### I am submitting a...

- [ ] Feature request
- [ ] Design defect
- [x] Source code defect
- [ ] Demo/documentation defect
- [ ] Other

#### charts version:

"@carbon/charts": "^0.40.14",
"@carbon/charts-angular": "^0.40.14",
"@carbon/icons-angular": "~11.1.4",
"@carbon/colors": "^10.5.0",
"@carbon/themes": "^10.16.0",
"carbon-addons-angular": "^1.4.1",
"carbon-components": "~10.18.0",
"carbon-components-angular": "^4.38.0",

### Issue description

We have integrated IBM Line Chart in one of our application and after running 'IBM Equal Access Accessibility Checker' tool  we are seeing below violation related to tooltip.

**"a class="tooltip-color tooltip-1-1-1 
Hyperlink has no link text, label or image with a text alternative"**

issue reference link - https://able.ibm.com/rules/tools/help/WCAG20_A_HasText

<img width="1252" alt="image" src="https://user-images.githubusercontent.com/78785953/107331635-2a983e00-6ad9-11eb-9f86-79bab5d5d3c5.png">

#### Steps to produce the issue

1. Go to Chrome and install below plugin

https://chrome.google.com/webstore/detail/ibm-equal-access-accessib/lkcagbfjnkomcinoddgooolagloogehp

2. Open below Carbon chart link

https://carbon-design-system.github.io/carbon-charts/angular/?path=/story/line--line-discrete

3. Right click and open inspect element

4. Click on Elements Tab

5. Select Accessibility checker and run the scan

We see many issue and out of that below is the one which is causing issue in our application

<img width="1648" alt="image" src="https://user-images.githubusercontent.com/78785953/107333328-54eafb00-6adb-11eb-8d9a-abf91ef7e31c.png">

#### Current behavior

Accessibility error is coming for anchor tab as above

#### Expected behavior

Accessibility error should not come.

So either need to add some text or need to follow below recommendations

<img width="1172" alt="image" src="https://user-images.githubusercontent.com/78785953/107333443-7fd54f00-6adb-11eb-9b13-e1aca49491dc.png">

### Screenshot or recording

Attached above
